pkg/kapis/iam/v1alpha2/handler.go

```diff
@@ -899,6 +908,14 @@ func (h *iamHandler) CreateNamespaceMembers(request *restful.Request, response *
 		}
 	}
 
+	if event := requestinfo.AuditEventFrom(request.Request.Context()); event != nil {
```

Comment:
Auditing happens all the operations are successful, but what if some of the members are created successfully, others are not? Don't we audit every MODIFY operation even there are errors?